src/components/Delete.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+
+class Delete extends Component {
```

Comment:
`Add` and `Delete` could be the same component to which you provide different text and action props.